cs251tk/student/markdownify/markdownify.py

```diff
@@ -4,15 +4,24 @@
 from collections import OrderedDict
 from .process_file import process_file
 from .find_warnings import find_warnings
+from ...common import chdir
+from cs251tk.common import run
 
 
 def markdownify(spec_id, *, username, spec, basedir, debug, interact):
     """Run a spec against the current folder"""
     try:
+        """Try to run the CheckDates program. If not installed, move on."""
+
+        with chdir(os.path.join(basedir, 'data')):
+            _, res, _ = run([os.path.join(basedir, 'data', 'CheckDates'), '-t', '-s', username, '-a', spec_id])
+        if res[:9] == "[Errno 2]":
```

Comment:
Can we find the proper Python `Exception` class and check for it here?

Reply:
like, a `try/except` clause thing

Reply:
Done. It's apparently an IOError.